### PR TITLE
Cronjob via packages

### PR DESCRIPTION
**Pull request by LaurenceJJones**

### CODE REVIEW

File: debian/postrm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 if [ "$1" = "purge" ]; then
    find /etc/crowdsec -maxdepth 1 -mindepth 1 | grep -v "bouncer" | xargs rm -rf || echo ""
    rm -rf /var/lib/crowdsec
-   if [ -d "/etc/cron.daily/" ] && [ -f "/etc/cron.daily/crowdsec-hub" ]; then
-      rm /etc/cron.daily/crowdsec-hub
+   if [ -d "/etc/cron.daily/" ] && [ -f "/etc/cron.daily/crowdsec" ]; then
```
Comment (mmetc) [inline debian/postrm:4]: the -d is redundant

File: rpm/SPECS/crowdsec.spec [hunk 2]
```diff
@@ -229,7 +233,7 @@ if [ $1 == 0 ]; then
 fi
 
 if [ -d "/etc/cron.daily/" ] && [ -f "/etc/cron.daily/crowdsec-hub" ]; then
```
Comment (LaurenceJJones) [inline rpm/SPECS/crowdsec.spec:235]: @mmetc Redundant here too?

**Review by mmetc — APPROVED** [review]

File: rpm/SPECS/crowdsec.spec [hunk 3]
```diff
@@ -52,6 +53,8 @@ mkdir -p %{buildroot}%{_sharedstatedir}/%{name}/plugins
 mkdir -p %{buildroot}%{_sysconfdir}/crowdsec/notifications/
 mkdir -p %{buildroot}%{_libdir}/%{name}/plugins/
 
+mkdir -p %{buildroot}%{_sysconfdir}/cron.daily
```
[the remark this answers is not in the document]
Comment (LaurenceJJones) [reply rpm/SPECS/crowdsec.spec]: Yes it does

File: debian/crowdsec.cron.daily [hunk 4]
```diff
@@ -0,0 +1 @@
+../config/crowdsec
```
[the remark this answers is not in the document]
Comment (LaurenceJJones) [reply debian/crowdsec.cron.daily]: The file is a symlink but git shows the contents of the symlink

**Review by sabban — APPROVED** [review]
lgtm: need testing when rpms are built

**Review by mmetc — APPROVED** [review]

**Pull request merged**